contents/en/basics/import.md

```diff
@@ -120,9 +120,18 @@ import {
 } from 'echarts/components';
 import { LabelLayout, UniversalTransition } from 'echarts/features';
 import { CanvasRenderer } from 'echarts/renderers';
+import type { 
+  ComposeOption, 
+  BarSeriesOption, 
+  LineSeriesOption, 
+  TitleComponentOption,
+  TooltipComponentOption,
+  GridComponentOption,
+  DatasetComponentOption
```

Comment:
Could you eleborate this change? It looks duplicated with the above imports.

https://github.com/apache/echarts-handbook/blob/3287aae4dcc945a5ff92401a3945f82449d6b270/contents/en/basics/import.md#L100-L120

Reply:
My newly added import is the type

Reply:
The `TitleComponentOption`, `TooltipComponentOption` etc. are already the type of the corresponding component. Is it necessary to import the types again? 🤔

Reply:
I seem to have misread before, but it seems that type and component are imported together. Perhaps it would be better to import separately, and it would be clearer to add the `type` keyword to the type import?

Reply:
> Perhaps it would be better to import separately, and it would be clearer to add the type keyword to the `type` import?

Yeah. I agree with you. It would be better if we do that.